jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/JmxScraperContainer.java

```diff
@@ -76,6 +76,7 @@ public void start() {
     // for now only configure through JVM args
     List<String> arguments = new ArrayList<>();
     arguments.add("java");
+    arguments.add("-Dotel.metrics.exporter=otlp");
```

Comment:
missing this one means the SDK does not export, we should probably set it by default when not set explicitly as it's the only option for exporting metrics.

Reply:
No, this does not change the default, it's only that whenever we run JmxScraper in this docker setup we need to export metrics through OTLP, so it's not worth making it configurable for now unless there are other use-cases.